packages/admin/admin/src/form/ChipSelect.tsx

```diff
@@ -0,0 +1,201 @@
+            padding: 0;
+            height: 100% !important;
+            min-height: unset !important;
+        }
+    `,
+);
+
+export interface ChipSelectProps<T = string>
+    extends ThemedComponentBaseProps<{
+            root: "div";
+            inputRoot: "div";
+            select: typeof Select;
+            menuItem: typeof MenuItem;
+            inputBase: typeof MuiInputBase;
+        }>,
+        Omit<ChipProps, "children" | "onChange"> {
+    children?: React.ReactNode;
+    getOptionLabel?: (option: T) => string;
+    getOptionValue?: (option: T) => string;
+    options?: T[];
+    selectedOption?: T;
+    onChange: (event: SelectChangeEvent) => void;
+    fullWidth?: boolean;
+}
+
+type ChipInputProps = InputBaseProps & {
+    chipProps?: ChipProps;
+    inputBaseProps?: InputBaseProps;
+    inputRootProps?: React.ComponentPropsWithoutRef<"div">;
+};
+
+const ChipInput = ({ chipProps, inputBaseProps, inputRootProps, ...restProps }: ChipInputProps) => (
+    <ChipInputRoot {...inputRootProps}>
+        <Chip
+            icon={<ChevronDown />}
+            label={<LabelInputBase {...inputBaseProps} {...restProps} />} // use second InputBase to show the label
+            variant="filled"
+            {...chipProps}
+        />
+        <InputBase {...inputBaseProps} {...restProps} />
+    </ChipInputRoot>
+);
+
+export function ChipSelect<T = string>(inProps: ChipSelectProps<T>) {
+    const {
+        options,
+        getOptionLabel = (option: T) => {
+            if (typeof option === "object" && option !== null) {
+                if ((option as any).label) return String((option as any).label);
+                if ((option as any).id) return String((option as any).id);
+                if ((option as any).value) return String((option as any).value);
+                return JSON.stringify(option);
+            } else {
+                return String(option);
+            }
+        },
+        getOptionValue = (option: T) => {
+            if (typeof option === "object" && option !== null) {
+                if ((option as any).value) return String((option as any).value);
+                if ((option as any).id) return String((option as any).id);
+                return JSON.stringify(option);
+            } else {
+                return String(option);
+            }
+        },
+        selectedOption,
+        children,
+        slotProps,
+        onChange,
+        fullWidth = false,
+        className,
+        ...chipProps
+    } = useThemeProps({
+        props: inProps,
+        name: "CometAdminChipSelect",
+    });
+
+    const ownerState = { fullWidth };
+
+    return (
+        <Root {...slotProps?.root} ownerState={ownerState}>
+            <Select
+                {...slotProps?.select}
+                value={selectedOption || ""}
+                onChange={onChange}
+                input={<ChipInput chipProps={chipProps} inputBaseProps={slotProps?.inputBase} inputRootProps={slotProps?.inputRoot} />}
```

Comment:
`chipProps` should also include the slots `slotProps`

```suggestion
                input={<ChipInput chipProps={{ ...chipProps, ...slotProps?.chip }} inputBaseProps={slotProps?.inputBase} inputRootProps={slotProps?.inputRoot} />}
```